src/Application/test/DivideCalculatorGTest.cpp

```diff
@@ -0,0 +1,75 @@
+///////////////////////////////////////////////////
+/**
+ * @file DivideCalculatorGTest.cpp
+ * @brief Unit tests for Divide Calculator
+ */
+///////////////////////////////////////////////////
+
+#include<gtest/gtest.h>
+#include<gmock/gmock.h>
+#include<limits>
+
+#include "CalculatorApplication.h"
+#include "CalculatorApplicationFactory.h"
+#include "CalculatorStrings.h"
+#include "DivideCalculator.h"
+
+using namespace calculator;
+using namespace testing;
+
+class whenTestingDivideCalculator : public :: testing :: Test
```

Comment:
If you aren't using the fixture... just remove this class and then change all of your `TEST_F(...)` to `TEST(...)`